modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/NavigationBar.js

```diff
@@ -25,25 +25,17 @@ export default function NavigationBar({cssClass, inverted, navigationItems}) {
 				inverted={inverted}
 				triggerLabel={navigationItems.find(({active}) => active)?.label}
 			>
-				{navigationItems.map(({active, href, label}, index) => {
-					const linkAttributes = href ? {href} : {};
-
-					return (
-						<ClayNavigationBar.Item
-							active={active}
-							data-nav-item-index={index}
```

Comment:
Sigh...

Reply:
FYI, I have no idea what you're talking about 😂 — this is what GitHub UI shows me here:

![Screenshot 2020-06-16 -123938-pfyKMTLY@2x](https://user-images.githubusercontent.com/7074/84764756-80f6c080-afce-11ea-8274-b5bf176448ce.png)